src/company_gui.cpp

```diff
@@ -159,15 +155,14 @@ static uint GetMaxCategoriesWidth()
  */
 static void DrawCategory(const Rect &r, int start_y, ExpensesList list)
 {
-	int offs_left = _current_text_dir == TD_LTR ? EXP_INDENT : 0;
-	int offs_right = _current_text_dir == TD_LTR ? 0 : EXP_INDENT;
+	const Rect tr = r.Indent(WidgetDimensions::scaled.hsep_indent, _current_text_dir == TD_RTL);
 
 	int y = start_y;
 	ExpensesType et;
 
 	for (uint i = 0; i < list.length; i++) {
 		et = list.et[i];
-		DrawString(r.left + offs_left, r.right - offs_right, y, STR_FINANCES_SECTION_CONSTRUCTION + et);
+		DrawString(tr.left, tr.right, y, STR_FINANCES_SECTION_CONSTRUCTION + et);
```

Comment:
Possible future work to change DrawString to take a Rect as well, rather than the left/right/top param?

Reply:
It does, but in this case y/start_y is provided separately. Although `tr.top = start_y;` would work, and then just increment tr.top instead of y.